### DISCUSSION TITLE
Error loading CSS file

### DISCUSSION THREAD

**Discussion by RobertoValente** [vercel/community#6251] [Help] [answered]
### Summary

Basically, on my localhost it works very well (im using wsl) and the src/pages/index.html found the file src/styles/stylec.css. When i deploy it to vercel and check the website, the style is not loaded.

I set the Root Directory to: src/pages
In index.html im calling the css file with: <link rel="stylesheet" href="../styles/style.css">
> 99% is a miss detail by my side, but i really didnt find out already

On the Example Link, will be my website to you see the error

### Example

https://robertovalente.pt

### Steps to Reproduce

About the steps to replicate it, i've already say it above,

**Comment by amyegan** [comment]
Hey @RobertoValente. One thing you can try is moving the CSS file into a `/public` directory at the root of the project. That will ensure it is available to your HTML file when deployed.

If that doesn't work please share more about your project to help us get you a good answer. Based on what you posted, I don't know what framework or build settings the project uses. 🙏

**Reply by RobertoValente** [reply, marked as answer]
Hi @amyegan , after see your reply i checked my code (with "refreshed" mind) and the problem was about the root directory that i've defined on vercel settings. I changed it, the path that i was calling the css file too and everything works! Thank you
